apps/governance/src/routes/home/index.tsx

```diff
@@ -136,6 +139,35 @@ const HomeNodes = ({
   );
 };
 
+interface SimpleValidatorRendererProps {
+  id: string;
+  avatarUrl: string | null | undefined;
+  name: string;
+}
+
+export const SimpleValidatorRenderer = ({
```

Comment:
nit (non-blocking): the naming feels weird here, overly verbose without actually telling much. including `simple` in the name usually makes sense when there are multiple variations, which is not the case here, if i'm interpreting this correctly. renderer is basically every react component, because that's just what they do :) I might consider something like `ValidatorItem`, `ValidatorLink`, etc, not sure if it makes any sense.
also not very important, feel free to dismiss please

Reply:
Thanks - there is a different validator renderer for the validator's table, but it's a poor naming choice. Why is naming so hard!? 😭